package.json

```diff
@@ -73,10 +73,10 @@
     "text-table": "^0.2.0"
```

Comment:
Could you update table in this as well? Latest version is 5.2.3 / they also have lodash as a dependency.